waspc/src/Wasp/Generator/ServerGenerator/JobGenerator.hs

```diff
@@ -101,9 +97,6 @@ genAllJobImports spec =
         [ "name" .= jobName
         ]
 
-relPosixPathFromJobFileToExtSrcDir :: Path Posix (Rel ServerSrcDir) (Dir GeneratedExternalCodeDir)
```

Comment:
It's not really needed IMHO. 

When you use `getJsImportStmtAndIdentifier` from the `ServerGenerator` it's generating an import statement with a path relative to `GeneratedExternalCodeDir` in `ServerSrcDir` anyways.

That's also encoded in the first argument's type `PathFromImportLocationToExtCodeDir` "just tell me a relative path".

So, this whole definition is not necessary, we could only encode it with a definition like `relPosixPathFromJobFileToExtSrcDir = [reldirP|../|]` but it feels kinda redundant since that's what the first argument encodes anyways. Let me know what do you think.